scripts/binary/util/upload.js

```diff
@@ -69,9 +69,12 @@ const getPublisher = async function () {
     params: {
       Bucket: S3Configuration.bucket,
     },
-    accessKeyId: aws.accessKeyId,
-    secretAccessKey: aws.secretAccessKey,
-    sessionToken: aws.sessionToken,
+    credentials: {
```

Comment:
Aww the missing piece to why this needed to be reverted: https://github.com/cypress-io/cypress/pull/28379